lib/utils/linter.js

```diff
@@ -0,0 +1,27 @@
+const log = require('./log')
+const wistLogger = require('./wist-logger')
+const CLIEngine = require('@willowtreeapps/wist').CLIEngine
+
+function run(sourceFiles, ignoreErrors) {
```

Comment:
This is a neat first step. I think the way forward from here would be to come up with a way to parallelize the linting process.